shared/metadata.go

```diff
@@ -4,6 +4,39 @@
 
 package shared
 
+import (
+	"net/http"
+	"path"
+	"sort"
+
+	"github.com/go-yaml/yaml"
+	"github.com/web-platform-tests/wpt-metadata/util"
+)
+
+// MetadataFlag determines whether Metadata Information returns along
+// with a test result query request.
+// TODO(kyleju): Remove this flag once feauture is launched.
+const MetadataFlag = "metadataInfo"
```

Comment:
I don't think you need to hide this behind a flag server-side - it's controlled by a flag on the client and is not a dangerous feature.

Reply:
This is the key value of Metadata param in the url. I have renamed it to MetadataKey to avoid confusions.